src/app/server/Dnssd.cpp

```diff
@@ -384,6 +384,10 @@ CHIP_ERROR DnssdServer::Advertise(bool commissionableNode, chip::Dnssd::Commissi
         }
     }
 
+    if (commissionableNode && mode != chip::Dnssd::CommissioningMode::kDisabled)
+    {
+        DeviceLayer::ConfigurationMgr().StartAdvertisement();
```

Comment:
```suggestion
        // notify device layer that advertisement is beginning (to do work such as increment rotating id)
        DeviceLayer::ConfigurationMgr().StartAdvertisement();
```

Reply:
The comment seems to be re-stating what the code says (It says "start advertisement" in device layer) and seems to somewhat imply implementation details like incrementing rotating id - rotating id may even be disabled.

Could we restate in some way that expands on code or just delete it if  we cannot say more? Maybe the naming is wrong and it should be `NotifyOfAdvertisementStart`

Reply:
Read the rest of the code. The name *is* wrong hence the need of the comment. Please fix the name instead.